articles/native-platforms/android/02-custom-login.md

```diff
@@ -44,23 +43,24 @@ Then, run "Sync project with Gradle files".
 
 You need to add the following permissions inside the ``AndroidManifest.xml``:
         
+```xml
 	<uses-permission android:name="android.permission.INTERNET" />
```

Comment:
@tenmaster Please remove excessive tabs/spaces at the beginning of line. This comment goes to all scripts in this doc, please have a look, and check also the alignment of brackets `{}`, at some it's off. Thanks.